SpatialGDK/Source/SpatialGDK/Public/Utils/SpatialStatics.h

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "Kismet/BlueprintFunctionLibrary.h"
+#include "SpatialStatics.generated.h"
+
+UCLASS()
+class SPATIALGDK_API USpatialStatics : public UBlueprintFunctionLibrary
+{
+    GENERATED_BODY()
+
+private:
+	
+	static bool IsSpatialOffloadingEnabled();
+
+	static class UActorGroupManager* GetActorGroupManager(const UObject* WorldContext);
+
+public:
+
+	/**
+	 * Returns true if SpatialOS Networking is enabled.
+	 */
+	UFUNCTION(BlueprintPure, Category = "SpatialOS")
+	static bool IsSpatialNetworkingEnabled();
+
+	/**
+     * Returns true if the current Worker Type owns the Actor Group this Actor belongs to.
+	 * Equivalent to HasAuthority when Spatial Networking is disabled.
+     */
+    UFUNCTION(BlueprintPure, Category = "SpatialOS|Offloading")
+    static bool IsActorGroupOwnerForActor(const AActor* Actor);
+
+	/**
+	 * Returns true if the current Worker Type owns the Actor Group this Actor Class belongs to.
+	 * Equivalent to HasAuthority when Spatial Networking is disabled.
+	 */
+	UFUNCTION(BlueprintPure, Category = "SpatialOS|Offloading", meta = (WorldContext = "WorldContextObject"))
+	static bool IsActorGroupOwnerForClass(const UObject* WorldContextObject, const TSubclassOf<AActor> ActorClass);
+
+	/**
+	 * Returns true if the current Worker Type owns this Actor Group.
+	 * Equivalent to HasAuthority when Spatial Networking is disabled.
```

Comment:
Is that completely equivalent? This seems like a completely new concept. Right now it seems slightly confusing because the worker can own the actor group but also not have authority. I think we should keep the abstraction that HasAuthority corresponds to HasAuthority and that this is new.

Reply:
Resolved Below.